### DISCUSSION TITLE
install MSI not MSIX

### DISCUSSION THREAD

**Discussion by rampd2** [microsoft/winget-cli#2892] [Q&A]
How to install (PowerShell) with a msi package instead of a msix bundle on x64 currently?

Even [Microsoft Docs - Install PowerShell on Windows (inside a "Note")](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.3#winget) state:

> On Windows systems using X86 or X64 processor, winget installs the MSI package.
> On systems using the Arm64 processor, winget install the Microsoft Store (MSIX) package.
> For more information, see Installing from the Microsoft Store.

It installs the MSIX variant anyways but that comes with some Problems as described at the top of the document:
> - Microsoft Store package - An easy way to install for casual users of PowerShell **but has limitations**

For PowerShell specific it got changed with this pull request <https://github.com/microsoft/winget-pkgs/pull/91378>, [here](https://github.com/microsoft/winget-pkgs/pull/91378/commits/3bfe6a0a14a02f6b2f1abff1ed74910f71704e0a#diff-643658e4110a0c006b9c0c48afd270938e81e57b7b10b4c97f1d04a5fb512c19R31-R36).

I could only find issue #1166 which is not really helping as it's currently open.

**Comment by denelon** [comment]
Related to:
* https://github.com/microsoft/winget-pkgs/issues/94861

There was an unexpected change to the manifest.

**Reply by rampd2** [reply]
So as I understand the currently best/newest information is that [these MSIX bugs are assigned to an engineer to fix](https://github.com/microsoft/winget-pkgs/issues/94861#issuecomment-1414514424) and there seems to be a workaround to use the flag `--scope=machine` (not tested by me) to force msi installation.

Good to see it going forward.

Since this is not really solved I will not yet mark your suggested answer as answer.

_Sorry for the "late" reply, GitHub somehow didn't notify me (enough)._
